dev-haskell/bibtex/bibtex-0.1.0.6.ebuild

```diff
@@ -17,7 +17,7 @@ SLOT="0/${PV}"
 KEYWORDS="~amd64 ~x86"
 #IUSE="buildexamples"
 
-RESTRICT="buildexamples" #requires unacceptably old verions
+# RESTRICT="buildexamples" #requires unacceptably old verions
```

Comment:
I suggest removing incorrect valie completely instead of keeping an invalid comment.

Reply:
No problem - the only reason I left it was to follow the same style used in previous commits. I'll remove all the commented lines.

Reply:
nvm, I'll just delete the commented RESTRICT line.